doc/source/data/working-with-text.rst

```diff
@@ -0,0 +1,131 @@
+Working with Text
+=================
+
+Text data is ubiquitous in use cases like Natural Language Processing.
```

Comment:
Could use recommendations on a better introductory sentence

Reply:
With Ray Data you can easily read and transform large datasets consisting of text data.